include/HypreDirectSolver.h

```diff
@@ -93,28 +93,28 @@ class HypreDirectSolver: public LinearSolver
 
   mutable HYPRE_Solver precond_;
 
-  HypreIntType (*solverCreatePtr_)(MPI_Comm, HYPRE_Solver*);
-  HypreIntType (*solverDestroyPtr_)(HYPRE_Solver);
-  HypreIntType (*solverSetupPtr_)(
+  linSysIntType (*solverCreatePtr_)(MPI_Comm, HYPRE_Solver*);
```

Comment:
We cannot replace `HypreIntType` with Tpetra's GlobalOrdinal here. None of the return types or arguments should be changed

Reply:
I'll revise and re-test. I thought Hyper was turned on in this build, apparently not...